server/templates/staff/course/students.list.html

```diff
@@ -37,6 +38,11 @@ <h3 class="box-title"><span> {{ title }} </span></h3>
                     <td class="role"><span class="label label-success">{{item.role}}</span></td>
                     <td class="timestamp" data-timestamp="{{ item.created }}">{{ utils.local_time(item.created, current_course) }}</td>
                     <td class="section">{{item.section}}</td>
+                    <td class="unenroll">
+                        {% call forms.render_form(unenroll_form, action_url=url_for('.unenrollment', cid=current_course.id, user_id=item.user.id), action_text='Unenroll',
```

Comment:
The traditional action button takes a lot of space. I'd prefer a smaller button with an icon. 

If you use `render_form_bare` you can get rid of the action_text and specify your own submit button inside of the call block. 

Here's what I had disabled before - I think it worked pretty well. 
` <button type="submit" class="btn btn-sm btn-danger"> <i class="fa fa-trash-o"></i></button>`
![image](https://cloud.githubusercontent.com/assets/882381/18031436/6db196b0-6c95-11e6-89cd-a480db178138.png)